{"name": "test_uptate_kf[args1]", "status": "passed", "steps": [{"name": "执行接口", "status": "passed", "start": 1715834327706, "stop": 1715834328070}, {"name": "断言响应体", "status": "passed", "steps": [{"name": "断言：errcode是否与预期结果一致", "status": "passed", "start": 1715834328070, "stop": 1715834328070}, {"name": "断言：errmsg是否与预期结果一致", "status": "passed", "start": 1715834328070, "stop": 1715834328070}], "start": 1715834328070, "stop": 1715834328070}], "attachments": [{"name": "log", "source": "9e0c0402-6ef3-4aec-810e-5aaaa33abcf2-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "args", "value": "{'caseName': '【设置客服信息接口】用例2：客服账号kf_account参数为空', 'method': 'post', 'url': '/customservice/kfaccount/update', 'data': {'kf_account': '', 'nickname': 'yrEb6drY2MJ3IiRU'}, 'excepted': {'errcode': 65401, 'errmsg': 'invalid custom service account', 'statusCode': 200}}"}], "start": 1715834327706, "stop": 1715834328074, "uuid": "2892bb91-12fe-4fbe-9dda-689ca430ddbd", "historyId": "6aaecb4311ef356632d0bdb49b8b01dc", "testCaseId": "3d649fc028268d23ad4ea1e03c8ffc07", "fullName": "testcase.kf_mannage.test_kf.Test_kf#test_uptate_kf", "labels": [{"name": "feature", "value": "客服管理模块"}, {"name": "story", "value": "设置客服信息接口"}, {"name": "epic", "value": "微信项目"}, {"name": "parentSuite", "value": "testcase.kf_mannage"}, {"name": "suite", "value": "test_kf"}, {"name": "subSuite", "value": "Test_kf"}, {"name": "host", "value": "DESKTOP-IG8PJM0"}, {"name": "thread", "value": "4692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.kf_mannage.test_kf"}]}